### PR TITLE
ci: also run javascript linter

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by thisconnect** [comment]
in `frontend/web`:

it seems to miss eslint, make sure `npm install`(or `yarn install`) did run first

would be great to have `npm list` (or `yarn list`) after the install to see the module tree

**Comment by benma** [comment]
AppVeyor only builds the windows package, but does not run the tests/linters.

File: Makefile [hunk 1]
```diff
@@ -52,6 +52,7 @@ generate:
 webdev:
 	make -C frontends/web dev
 weblint:
+	yarn --cwd=${WEBROOT} install
```
Comment (x1ddos) [inline Makefile]: but you already added this command to scripts/ci.sh before running "make weblint", plus it is also run in "make generate", which you need to execute before any webdev, right.

maybe don't add yarn install here at all?

Comment (thisconnect) [reply Makefile]: Could `envinit` be the place to do `yarn install`?

Comment (benma) [reply Makefile]: No, `envinit` installs stuff to the host system (not good, should be improved some day). `yarn install` is local to the repo.

Comment (benma) [reply Makefile]: @x1ddos good catch. Was a leftover :scream:

**Pull request merged**